FLAnimatedImage/FLAnimatedImageView.m

```diff
@@ -308,10 +308,15 @@ - (void)startAnimating
             [self.displayLink addToRunLoop:[NSRunLoop mainRunLoop] forMode:self.runLoopMode];
         }
 
-        // Note: The display link's `.frameInterval` value of 1 (default) means getting callbacks at the refresh rate of the display (~60Hz).
+        // Note: The display link's `.frameInterval` value of 1 (default) means getting callbacks at the refresh rate of the display.
         // Setting it to 2 divides the frame rate by 2 and hence calls back at every other display refresh.
-        const NSTimeInterval kDisplayRefreshRate = 60.0; // 60Hz
-        self.displayLink.frameInterval = MAX([self frameDelayGreatestCommonDivisor] * kDisplayRefreshRate, 1);
+        const NSTimeInterval kStaticDisplayRefreshRate = 60.0;
+
+        if (@available(iOS 10.3, *)) {
+            self.displayLink.frameInterval = MAX([self frameDelayGreatestCommonDivisor] * (double) UIScreen.mainScreen.maximumFramesPerSecond, 1); // 60Hz or 120Hz, depending on device capabilities.
```

Comment:
On iOS 10.3 and above, read the available maximum frame rate directly from UIScreen.